gitlab-runners/ubi7-gitlab-runner/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM registry.access.redhat.com/ubi7
+
+LABEL maintainer="Red Hat Services"
+
+ENV HOME /home/gitlab-runner
+WORKDIR /home/gitlab-runner
+
+# add user
+RUN adduser --create-home --user-group gitlab-runner \
+  && chgrp -Rf root $HOME \
+  && chmod -Rf g+w $HOME \
+  && chmod g+w /etc/passwd
+
+RUN update-ca-trust
+
+# Update image
+RUN yum update -y && rm -rf /var/cache/yum
+
+# Add GitLab’s official repository
+RUN curl -L https://packages.gitlab.com/install/repositories/runner/gitlab-runner/script.rpm.sh | bash
+
+# Install packages
+RUN yum install -y \
+  gitlab-runner \
+  && yum clean all
+
+# install entrypoint
+COPY entrypoint /
+RUN chmod +x /entrypoint \
+    && chown gitlab-runner /entrypoint
+
+USER gitlab-runner
+
+STOPSIGNAL SIGQUIT
+VOLUME ["/etc/gitlab-runner", "/home/gitlab-runner"]
+ENTRYPOINT ["/usr/bin/dumb-init", "/entrypoint"]
```

Comment:
`dumb-init` seems to be missing
```
podman run --rm localhost/glr --help                                                                                                                                 
Error: executable file not found in $PATH: No such file or directory: OCI runtime command not found error
podman run --rm --entrypoint bash localhost/glr -c 'ls /usr/bin/dumb-init'
ls: cannot access /usr/bin/dumb-init: No such file or directory
```

Reply:
this was a merge of @sabre1041 work and https://gitlab.com/gitlab-org/gitlab-runner/-/blob/master/dockerfiles/alpine/Dockerfile

I never tried to run locally just in the context of the helm chart (https://docs.gitlab.com/runner/install/kubernetes.html).

With my updates i just tried:
```
podman run ca9be29097f8
ERRO[0003] could not find slirp4netns, the network namespace won't be configured: exec: "slirp4netns": executable file not found in $PATH
Runtime platform                                    arch=amd64 os=linux pid=8 revision=ce065b93 version=12.10.1
Starting multi-runner from /home/gitlab-runner/.gitlab-runner/config.toml...  builds=0
WARNING: Running in user-mode.
WARNING: Use sudo for system-mode:
WARNING: $ sudo gitlab-runner...

Configuration loaded                                builds=0
listen_address not defined, metrics & debug endpoints disabled  builds=0
[session_server].listen_address not defined, session endpoints disabled  builds=0
ERROR: Failed to load config stat /home/gitlab-runner/.gitlab-runner/config.toml: no such file or directory  builds=0
ERROR: Failed to load config stat /home/gitlab-runner/.gitlab-runner/config.toml: no such file or directory  builds=0
ERROR: Failed to load config stat /home/gitlab-runner/.gitlab-runner/config.toml: no such file or directory  builds=0
ERROR: Failed to load config stat /home/gitlab-runner/.gitlab-runner/config.toml: no such file or directory  builds=0
ERROR: Failed to load config stat /home/gitlab-runner/.gitlab-runner/config.toml: no such file or directory  builds=0
^CWARNING: Graceful shutdown not finished properly    builds=0 error=received: interrupt
WARNING: Starting forceful shutdown                 StopSignal=interrupt builds=0
All workers stopped. Can exit now                   builds=0
```

looks right to me because it requires a config.toml which typically gets provided via a configmap

Reply:
That looks better and I now get the expected output as well.